docs/unraid-os/advanced-configurations/optimize-storage/zfs-storage.md

```
@@ -0,0 +1,383 @@
+- ZFS send/receive for efficient replication  
+- On-the-fly compression  
+
+Unraid supports ZFS for any storage pool. You can create a new ZFS pool, import one from another system, or use Unraid’s unique hybrid ZFS setup: add a ZFS-formatted disk directly to the Unraid **array** (not a pool) and combine ZFS features with Unraid’s **parity** protection.
+
+:::info Example 
+You can use ZFS snapshots and replication on a single disk as a backup target, or replicate a fast SSD ZFS pool to a ZFS disk in the array protected by Unraid parity - getting the best of both worlds.
+:::
+
+<div style={{ margin: 'auto', maxWidth: '600px', display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
+
+![](/img/zfs1.png)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)